migrator/migrations/m_170_to_m_171_create_policy_categories_and_edges/migration.go

```diff
@@ -0,0 +1,138 @@
+package m170tom171
+
+import (
+	"context"
+	"strings"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	frozenSchema "github.com/stackrox/rox/migrator/migrations/frozenschema/v74"
+	"github.com/stackrox/rox/migrator/migrations/loghelper"
+	policyCategoryEdgePostgresStore "github.com/stackrox/rox/migrator/migrations/m_170_to_m_171_create_policy_categories_and_edges/policycategoryedgepostgresstore"
+	policyCategoryPostgresStore "github.com/stackrox/rox/migrator/migrations/m_170_to_m_171_create_policy_categories_and_edges/policycategorypostgresstore"
+	policyPostgresStore "github.com/stackrox/rox/migrator/migrations/m_170_to_m_171_create_policy_categories_and_edges/policypostgresstore"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/uuid"
+	"gorm.io/gorm"
+)
+
+var (
+	startSeqNum = 170
+
+	migration = types.Migration{
+		StartingSeqNum: startSeqNum,
+		VersionAfter:   &storage.Version{SeqNum: int32(startSeqNum + 1)}, // 171
+		Run: func(databases *types.Databases) error {
+			err := CreatePolicyCategoryEdges(databases.GormDB, databases.PostgresDB)
+			if err != nil {
+				return errors.Wrap(err, "updating policy categories schema")
+			}
+			return nil
+		},
+	}
+
+	batchSize = 500
+	log       = loghelper.LogWrapper{}
+)
+
+type PolicyIDAndCategories struct {
```

Comment:
I think these can be removed since they are in  `policy_category_edges`

Reply:
Aah yes, remnants from pre winter break migration code that I was working on.